pkg/analyzer/analyzers/mailgun/mailgun.go

```diff
@@ -22,6 +19,15 @@ type Analyzer struct {
 	Cfg *config.Config
 }
 
+type SecretInfo struct {
```

Comment:
As we have now Token Information and Domain Information. Using SecretInfo like other analyzers make more sense.